src/utils/proceedPeople.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  Person, Sex, SearchField, SortParam,
+} from '../types';
+
+const sexOrder = Object.values(Sex);
+
+function sortPeople(people: Person[], sortParam: string, order: number) {
+  switch (sortParam) {
+    case SortParam.Name:
+      return people.sort((a, b) => order * a.name.localeCompare(b.name));
+    case SortParam.Sex:
+      return people.sort((a, b) => order * (
+        sexOrder.indexOf(a.sex) - sexOrder.indexOf(b.sex)));
+    case SortParam.Born:
+      return people.sort((a, b) => order * (a.born - b.born));
+    case SortParam.Died:
+      return people.sort((a, b) => order * (a.died - b.died));
```

Comment:
```suggestion
    case SortParam.Born:
    case SortParam.Died:
      return people.sort((a, b) => order * (a[sortParam] - b[sortParam]));
```
You can combine the similar cases